src/string-parser/stringify-auction.ts

```diff
@@ -0,0 +1,37 @@
+import { stringifyCall } from '.';
+import { AuctionCall } from '../types';
+/**
+ * Converts a list of calls in a contested auction to a string separated by -
+ * @param auction List of calls in auction
+ * @returns String representation of auction
+ */
+function stringifyContestedAuction(auction: AuctionCall[]): string {
+	const stringAuction = [];
+	for (const call of auction) {
+		stringAuction.push(stringifyCall(call));
+	}
+	return stringAuction.join('-');
+}
+/**
+ * Converts a list of calls in an uncontested auction to a string separated by - with all passes inserted
+ * @param auction List of calls in auction
+ * @returns String representation of auction
+ */
+function stringifyUncontestedAuction(auction: AuctionCall[]): string {
+	const stringUnconstedAuction = stringifyContestedAuction(auction);
```

Comment:
TODO check if there is a final pass and strip it. It needs to take as an argument the result of `parseUncontestedAuction` and return its initial string. Check its an even length and all odd indices are passes.